timesketch/frontend-ng/src/components/ShareCard.vue

```diff
@@ -0,0 +1,237 @@
+<!--
+Copyright 2022 Google Inc. All rights reserved.
```

Comment:
2023 ;)

Reply:
Well, the PR is from before the new year :) So technically it was written in 2022.